data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/HistogramAggregateActionConfig.java

```diff
@@ -25,27 +26,34 @@ public class HistogramAggregateActionConfig {
     public static final String DURATION_KEY = "duration";
     public static final Set<String> validOutputFormats = new HashSet<>(Set.of(OutputFormat.OTEL_METRICS.toString(), OutputFormat.RAW.toString()));
 
+    @JsonPropertyDescription("Name of the field in the events the histogram generates.")
     @JsonProperty("key")
     @NotNull
     String key;
 
+    @JsonPropertyDescription("The units for the values in the key.")
```

Comment:
Is this time units?

Reply:
It can be anything. It is the name of the units. It could be "bytes", "MB" and so on.